examples/tfengine/modules/team.hcl

```diff
@@ -490,10 +490,30 @@ template "additional_iam_members" {
 
 # Kubernetes Terraform deployment. This should be deployed after the GKE Cluster has been deployed.
 template "kubernetes" {
-  recipe_path = "{{.recipes}}/deployment.hcl"
+  recipe_path = "{{.recipes}}/project.hcl"
   output_path = "./kubernetes"
 
   data = {
+    project = {
+      project_id = "{{.prefix}}-{{.env}}-apps"
+      exists     = true
+    }
+    resources = {
+      kubernetes_service_accounts = [{
+        name = "ksa-gke"
```

Comment:
name it `ksa` is probably enough. It's fairly obvious ksa is for gke.

Reply:
Done.